gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/web/internal/configureWebApplication.kt

```diff
@@ -77,8 +79,16 @@ internal fun configureWebApplication(
     }
 
     project.tasks.withType(IncrementalSyncTask::class.java) {
+        if (it.name.contains("wasmJs", ignoreCase = true)) {
+            it.dependsOn(unpackRuntime)
+            it.from.from(unpackedRuntimeDir)
+        }
```

Comment:
Can we leave small explanation on why we are actually doing this? 
The other question - any chance those "wasmJs" tasks have some narrower type we can rely on rather then IncrementalSyncTask

Reply:
refactored and added the comments. Thanks to @ilgonmic